cloudify_cli/commands/blueprints.py

```diff
@@ -315,6 +315,7 @@ def get(blueprint_id, logger, client, tenant_name):
         blueprint_dict['deployments'] = blueprint_deployments
         print_single(columns, blueprint_dict, 'Blueprint:', max_width=50)
     else:
+        modify_resource_labels([blueprint_dict])
```

Comment:
What does modify_resource_labels actually do? It sounds like it changes things before we display them to the user, which doesn't sound like something we should be doing. Is it just badly named?

Reply:
It's just serializing labels: the function we use in `cfy blu list` and `cfy dep list` (https://github.com/cloudify-cosmo/cloudify-cli/blob/1a71412c50b9ff1a35f366008133f74326ce0fe1/cloudify_cli/labels_utils.py#L8-L17)